src/DataTable/Table.tsx

```diff
@@ -318,6 +319,10 @@ function TableCell({children, scope, ...rest}: TableCellProps) {
   )
 }
 
+function TableCellPlaceholder({children}: TableCellProps) {
```

Comment:
`TabelCellProps` was surprising as the type for `TableCellPlaceholder` since it wouldn't be rendering a `<td>` and instead would be inside of a `<td>`. If this is the case, would it help to use a standalone type here that just has `children` defined on it?

```suggestion
type TableCellPlaceholderProps = React.ChildrenWithProps<{}>;

function TableCellPlaceholder({children}: TableCellPlaceholderProps) {
```

Reply:
Great point! Silly mistake on my end.